### ISSUE TITLE
Error Set Tenant on FlexibleEngine : BucketAlreadyExists

### ISSUE THREAD

**Issue by Jean-PhilippeGouin** [CS-SI/SafeScale#18]
Hi,

There is a issue while setting the tenant on the FlexibleEngine of Orange Cloud.

``` 
/broker tenant set safescale_orange.x156sq
2018/10/04 09:29:05 Tenant Set called
failed to create Object Container 0.safescale: BucketAlreadyExists: The requested bucket name is not available. The bucket namespace is shared by all users oftry again.
        status code: 409
``` 

# Edit : 
**Workaround :** 

Setting the environement variable SAFESCALE_METADATA_SUFFIX works and add the suffix to the bucket name.

**Comment by vplanche-cssi** [comment]
S3 Object Stores (at minimum) uses a global namespace for all buckets. Due to this, we can't use '0.safescale' as metadata bucket name for all SafeScale tenants using the same Cloud Provider.

The metadata name has to be contextualized with tenant specific information (domain name for OpenStack for example)

**Comment by vplanche-cssi** [comment]
Fixed with commit 4653f82c642b096f0af2d4bf14a356ad7a30760e
